azext_iot/sdk/dps/service/provisioning_service_client.py

```diff
@@ -72,13 +78,17 @@ def __init__(
         super(ProvisioningServiceClient, self).__init__(self.config.credentials, self.config)
 
         client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
-        self.api_version = '2021-10-01'
+        self.api_version = '2021-11-01-preview'
```

Comment:
The specs for this API version has not been merged to the official rest API specs repository https://github.com/Azure/azure-rest-api-specs/tree/main/specification/deviceprovisioningservices/data-plane , making its readiness ambiguous particularly to be merged as part of a release branch.

Reply:
Yeah, I had followed up with the DPS Team about this, they informed me that the APIs are in Stewardship Review. As discussed, we need to ensure the following before merging the PR:

1. The APIs are approved by the Stewardship team (PR: https://github.com/Azure/azure-rest-api-specs/pull/17871)
2. The API version is available in all cloud environments.